_projects/2023/100481433.Rmd

```diff
@@ -0,0 +1,279 @@
+---
+title: "The world is progressively embracing democracy"
+description: |
+  Chart Replication Project.
```

Comment:
Please provide a longer description, kind of an abstract (see projects from last year).